src/foam/graphics/TreeGraph.js

```diff
@@ -59,13 +58,21 @@
              c.childNodes[i].x = 0;
            }
            c.maxLeft = c.maxRight = 0;
+
+           
          }
          this.doLayout();
        }.bind(this));
      },
 
      function doLayout() {
-       if ( this.root ) { this.root.layout(); this.root.doLayout(); }
+       if ( this.root ) { 
```

Comment:
Unrolling to two lines isn't bad, but all these extra blank lines can be removed.

Reply:
Done.

Reply:
Done.